devices/sm8150-mtp

```diff
@@ -0,0 +1,20 @@
+{% set PROJECT = PROJECT|default("") %}
+{% extends PROJECT+"fastboot.jinja2" %}
+
+{% set DEPLOY_OS = DEPLOY_OS|default("oe") %}
+{% set ROOTFS_URL_COMP = ROOTFS_URL_COMP|default("gz") %}
+
+{% set pre_boot_command = false %}
+{% set pre_power_command = false %}
+{% set pre_os_command = false %}
+{% set rootfs_label = 'userdata' %}
+
+{% block device_type %}sm8150-mtp{% endblock %}
+
+{% block auto_login_commands %}
```

Comment:
This block should already be included from include/fastboot.jinja2 that gets included from projects/lt-qcom/fastboot.jinja2

Reply:
@roxell Ack, older template also fixed previous qcom boads.